spec/lib/vmdb/loggers_spec.rb

```diff
@@ -10,12 +10,144 @@ def in_container_env(example)
     ENV['CONTAINER'] = old_env
   end
 
+  shared_examples "has basic logging functionality" do |logger|
+    subject       { logger }
+    let(:logger1) { subject }
+    let(:logger2) { $container_log }
+
+    before do
+      allow($container_log.logdev).to receive(:write)
+    end
+    before(:all) do
```

Comment:
I think this before(:all) is not necessary, because we don't run the tests with ENV["CONTAINER"] set.

Reply:
Oh, nvm. I completely misread this because of the shared examples being *passed* a logger.   The structure of the test is confusing in that respect.  Perhaps if `logger` is just replaced with a differently named variable like `test_logger` or something, then it is apparent it's not any built-in logger (which is how I read it).

Reply:
Yeah, `logger` is coming in from the shared example bit.  Changing it to `test_logger` or `logger_being_tested` (probably the former) is a simple enough change.  That said I would probably have to change it in the tests I added in https://github.com/ManageIQ/manageiq/pull/17223 first.